server/sonar-server/src/main/java/org/sonar/server/issue/notification/MyNewIssuesEmailTemplate.java

```diff
@@ -47,10 +47,11 @@ protected void appendAssignees(StringBuilder message, Notification notification)
   }
 
   @Override
-  protected String subject(Notification notification, String projectName) {
-    return String.format("You have %s new issues on project %s",
-      notification.getFieldValue(Metric.SEVERITY + COUNT),
-      projectName);
+  protected String subject(Notification notification, String projectName, @Nullable String branchName) {
+    return String.format("You have %s new issues on project %s%s",
+      notification.getFieldValue(Metric.RULE_TYPE + COUNT),
+      projectName,
+      branchName == null || branchName.isEmpty() ? "" : " (" + branchName + ")");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add parentheses to make the operator precedence explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS864)